go.mod

```diff
@@ -15,10 +15,10 @@ require (
 	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/afero v1.5.1
 	gotest.tools/v3 v3.0.3 // indirect
-	helm.sh/helm/v3 v3.3.4
-	k8s.io/api v0.18.9
-	k8s.io/apimachinery v0.18.9
-	k8s.io/client-go v0.18.9
+	helm.sh/helm/v3 v3.4.2
+	k8s.io/api v0.19.4
```

Comment:
The latest helm is using 0.19.4 but integration testing has passed. 
@rossf7 Can we go with this after adding changelog ?

Reply:
@tomahawk28 Can we also test it in chart-operator? 

If that's OK then I think it's fine to update.

Reply:
Good call, I will test on chart-operator.